packages/design-system-tokens/src/themes/healthcare/defaultTheme.ts

```diff
@@ -135,6 +135,29 @@ const components = {
   'badge__background-color--warn': themeColors.warn,
   'badge__font-color--warn': themeColors.base,
   'badge__border-radius': radius.pill,
+  // choice + choicelist
+  'choice__background-color': themeColors.background,
+  'choice__background-color--checked': themeColors.primary,
+  'choice__background-color--disabled': themeColors['gray-lighter'],
+  'choice__background-color--inverse': 'transparent',
+  'choice__background-color--disabled--inverse': hexOpacity(themeColors['muted-inverse'], 15),
+  'choice__background-color--unchecked': themeColors.primary,
```

Comment:
Same question as above - curious that `--checked` and `--unchecked` receive the same value

Reply:
Actually these unchecked variables are unused, will remove, was a part of the audit that was identified as an opportunity but we can always add them if needed.